torchtext/transforms.py

```diff
@@ -168,3 +168,24 @@ def forward(self, input: Any) -> Any:
     @property
     def label_names(self) -> List[str]:
         return self._label_names
+
+
+class Truncate(Module):
+    r"""Truncate input sequence
+
+    :param max_seq_len: The maximum allowable length for input sequence
+    :type max_seq_len: int
+    """
+
+    def __init__(self, max_seq_len: int) -> None:
+        super().__init__()
+        self.max_seq_len = max_seq_len
+
+    def forward(self, input: Any) -> Any:
+        """
+        :param input: Input sequence or batch of sequence to be truncated
+        :type input: Union[List[Union[str, int]], List[List[Union[str, int]]]]
```

Comment:
I think it some users could get confused why the annotation in code is different from docstring. Explaining it will help resolve such confusion. Also it will give opportunity to explain that there are connections between input types and output types.

Reply:
I think it's a good point. I will think through it and update the docs accordingly.